src/utils/IdealGraphVisualizer/pom.xml

```diff
@@ -77,8 +77,8 @@
                             <configuration>
                                 <rules>
                                     <requireJavaVersion>
-                                      <version>[11,18)</version>
-                                      <message>IGV requires a JDK version between 11 and 17</message>
+                                      <version>17</version>
```

Comment:
I suggest being a bit more conservative (safe) here and specifying an upper bound for the JDK version, since we cannot guarantee that all IGV dependencies will be compatible with any future JDK release. I suggest sticking to the newest JDK supported by the NetBeans Platform (for NetBeans 20 that would be JDK 21, see https://netbeans.apache.org/front/main/download/nb20/).

Reply:
ok, I changed it back to be JDK 17-21